README.md

```diff
@@ -30,28 +69,23 @@ build_type=Release
 compiler.cppstd=20
 ```
 
-## Using artifactory (temporary packages)
+### Artifactory
+
+1. Make sure artifactory is setup with Conan
 ```sh
 conan remote add --insert 0 conan-non-prod http://18.143.149.228:8081/artifactory/api/conan/conan-non-prod
 ```
-Now you should be able to download prebuilt `xrpl` package on some platforms. At the very least you should be able to skip steps 1 - 3 and jump to step 4 of the local package instructions for `rippled` (as described below) and conan should be able to fetch it from artifactory instead.
+Now you should be able to download prebuilt `xrpl` package on some platforms.
 
-## Preparing local packages (alternative to artifactory)
-1. Get rippled from [this branch](https://github.com/thejohnfreeman/rippled/tree/clio)
 2. Remove old packages you may have cached: 
 ```sh 
-conan remove -f xrpl/1.12.0-b2
-conan remove -f cassandra-cpp-driver/2.16.2
-```
-3. In a clone of rippled from step 1, setup rippled as per their instructions. Note that there is no need to build rippled, only make it available to Conan. Pay attention to these commands:
-```sh
-conan export external/snappy
-conan export external/soci
-conan export .
+conan remove -f xrpl
+conan remove -f cassandra-cpp-driver
 ```
-this will export a local package `xrpl/1.12.0-b2`.
 
-4. Navigate to clio's root directory and perform
+## Building Clio
+
+Navigate to Clio's root directory and perform
 ```sh
 conan export external/cassandra # export our "custom" cassandra driver package
```

Comment:
cassandra dependency seems available at remote artifactory . The local one may not be used actually

Reply:
Good point, it's on our artifactory already so should be sufficient. I'm a bit afraid that there are other dependencies with the same name (https://conan.io/center/recipes/cassandra-cpp-driver?version=2.15.3) but it does look like it only would resolve the older version (we have 2.16.3) so it should be forced to use ours (especially if artifactory was setup correctly with the `-insert 0` option).